process_pr.py

```diff
@@ -1003,6 +1036,9 @@ def process_pr(repo_config, gh, repo, issue, dryRun, cmsbuild_user=None, force=F
                     pull_request_updated = True
             continue
 
+        if (commenter == cmsbuild_user) and re.match(CMSBOT_TECHNICAL_MSG, first_line):
```

Comment:
also check if `technical_comment` is not already set. may be merge this block with the one above

Reply:
Will look into it.